package.json

```diff
@@ -97,6 +97,9 @@
     },
     "./codecs/raw": {
       "import": "./src/codecs/raw.js"
+    },
+    "./interface": {
+      "import": "./src/interface.js"
```

Comment:
This is so that one could import and use all the types as follows:

```ts
import * as API from 'multiformats/interface.js'

/**
 * @param {API.CID} cid
 */
export const test = (cid) => {
   // ...
}
```

Reply:
is there a case for exporting `./cid/interface` since it's going to be quite common that this is all you want?